documentation/asciidoc/computers/config_txt/boot.adoc

```diff
@@ -25,13 +25,15 @@ The Raspberry Pi 5 firmware defaults to loading `kernel_2712.img` because this i
 
 === `arm_64bit`
 
+NOTE: This flag has been removed from Raspberry Pi 5, as Raspberry Pi 5 only supports a 64-bit kernel.
```

Comment:
```suggestion
```

I suggest removing this because notes are disruptive to section flow and I think I've found a better way to convey this information in the text below. In particular, while calling out this specific model _might_ be helpful right now, it's not very futureproof: as we release more models, we'll have to expand this note to a massive and unhelpful size. Better to clarify in the meat and potatoes of this section, I think.